### ISSUE TITLE
Debug / tune the positioning model's SGD system

### ISSUE THREAD

**Issue by chris-jenkins-isi** [isi-vista/adam#617]
For scene types currently supported by the visualization system, success in positioning still comes down to luck, as objects can and will get stuck in sub-optimal locations. 

A few avenues to explore:
- relative weighting of each constraint in loss function
- model parameters/setup: set iterations vs. convergence, restarts, learning rate, etc
- changes to loss functions of each constraint to make them friendlier to SGD.
